src/components/common/Alpha.jsx

```diff
@@ -76,17 +72,6 @@ class Alpha extends ReactCSS.Component {
     }
   }
 
-  handleMouseDown(e) {
-    this.handleChange(e, true);
-    window.addEventListener('mousemove', this.handleChange);
```

Comment:
But I'll try - basically it's so that if you mouseDown in the div, then move your mouse outside of the div and mouseUp, when you move back into the div it won't still be dragging.